pkg/steps/multi_stage/init.go

```diff
@@ -123,15 +134,61 @@ func (s *multiStageTestStep) setupRBAC(ctx context.Context) error {
 			Subjects:   subj,
 		},
 		{
-			ObjectMeta: meta.ObjectMeta{Namespace: s.jobSpec.Namespace(), Name: "test-runner-view-binding", Labels: labels},
+			ObjectMeta: meta.ObjectMeta{Namespace: ns, Name: "test-runner-view-binding", Labels: labels},
 			RoleRef:    rbacapi.RoleRef{Kind: "ClusterRole", Name: "view"},
 			Subjects:   subj,
 		},
 	}
-
+	if s.vpnConf != nil {
+		bindings = append(bindings, rbacapi.RoleBinding{
+			ObjectMeta: meta.ObjectMeta{Namespace: ns, Name: s.name + "-vpn"},
+			RoleRef: rbacapi.RoleRef{
+				Kind: "ClusterRole",
+				Name: "ci-operator-vpn",
```

Comment:
Is this something to be defined in release repo?

Reply:
Yes, see the thread here: https://github.com/openshift/ci-tools/pull/2799#discussion_r862841042.

Reply:
Ack.